web/satellite/src/utils/plugins/segment.ts

```diff
@@ -0,0 +1,53 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+import loadScript from 'load-script';
+
+export class Segmentio {
```

Comment:
it will be super cool if types and public methods/fields will have some typedoc comments

Reply:
Added a comment for the class